modifications/SignatureVerifier.cc

```diff
@@ -0,0 +1,50 @@
+#include "SignatureVerifier.h"
+
+bool SignatureVerifier::convertJwkToPem(const std::string &strnn, const std::string& stree, const std::string& kid, std::string& strPubKey)
+{
+    auto nn = cppcodec::base64_url_unpadded::decode(strnn);
+    auto ee = cppcodec::base64_url_unpadded::decode(stree);
+    auto dd = cppcodec::base64_url_unpadded::decode(kid);
+
+    BIGNUM * modul = BN_bin2bn(nn.data(),nn.size(), NULL);
+    BIGNUM * expon = BN_bin2bn(ee.data(),ee.size(), NULL);
+    BIGNUM * kids   = BN_bin2bn(dd.data(),dd.size(), NULL);
+    RSA * rr = RSA_new();
+    EVP_PKEY * pRsaKey = EVP_PKEY_new();
+    RSA_set0_key(rr, modul, expon , kids);
+    EVP_PKEY_assign_RSA(pRsaKey, rr);
+    unsigned char * ss = new unsigned char[1024];
```

Comment:
any way of doing this without new delete and using a unique_ptr? 

@rhythmize

Reply:
A follow up question: Why are we writing this `SignatureVerifier::convertJwkToPem` conversion ourselves and not use jwt-cpp approach since we're already using jwt-cpp? https://github.com/Thalhammer/jwt-cpp/blob/master/example/jwks-verify.cpp
https://github.com/Thalhammer/jwt-cpp/blob/142d2d38e7bb1ad17b9787d776589e719d7494cb/include/jwt-cpp/jwt.h#L462